openapi/src/parts/controlled_azure_disk.yaml

```
@@ -24,6 +24,24 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{workspaceId}/resources/controlled/azure/disks/create-result/{jobId}:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+      - $ref: '#/components/parameters/JobId'
+    get:
+      summary: Retrieve information about an Azure Disk create job.
+      operationId: getCreateAzureDiskResult
+      tags: [ ControlledAzureResource ]
+      responses:
+        '200':
+          $ref: '#/components/responses/CreateControlledAzureDiskResponse'
+        '202':
+          $ref: '#/components/responses/CreateControlledAzureDiskResponse'
```

```suggestion
        '200':
          $ref: '#/components/responses/CreateControlledAzureResourceResponse'
        '202':
          $ref: '#/components/responses/CreateControlledAzureResourceResponse'
```
I am curious if we want to follow [this pattern](https://github.com/DataBiosphere/terra-workspace-manager/blob/67e624e81cadc5a975c04aab7661f1047a6d5d82/openapi/src/parts/controlled_azure_disk.yaml#L101-L104) re: naming the response.

Thoughts? (cc: @aherbst-broad)

---

@cpate4 yes, we haven't enforced this but following the defacto pattern would be good.

---

Hmm... they do differ slightly in the sense that they echo back the resource, with some additional fields that aren't on the request. Namely `AzureDiskCreationParameters` is transformed into the WSM-internal `AzureDiskResource` in the response.

I can see an argument that this echoing doesn't provide much value, but I would like this to be a separate ticket since it involves non-trivial testing on resources this PR doesn't touch and is technically a breaking API change insofar that it removes fields from a response body.

---

I did it for disk creation, but not any other resource creation